### DISCUSSION TITLE
Running scripts from CLI

### DISCUSSION THREAD

**Discussion by jscooksey** [netbox-community/netbox#8545] [Q&A] [answered]
I have scripts stored in the /opt/netbox/netbox/scripts path, and they are available and run from the web portal.
If I try to run them from the CLI though I get an error looking for the manage.py file:

`root@netbox:/opt/netbox/netbox/scripts# python3 manage.py runscript reserved_vlans.SearchVLANReserved
python3: can't open file '/opt/netbox-3.0.10/netbox/scripts/manage.py': [Errno 2] No such file or directory`

**Comment by kkthxbye-code** [comment, marked as answer]
`cd ..`

Then run the same command.

**Reply by candlerb** [reply]
Or when you're in the scripts directory:

```
python3 ../manage.py runscript reserved_vlans.SearchVLANReserved
```
